### PR TITLE
Initial commit to add site content.

### PR DESCRIPTION
**Pull request by ei8fdb**
I'm starting to add the content we spoke about on Friday. Issue #1 is tracking this work.

### CODE REVIEW

**Comment by ei8fdb** [comment]
Superceeded by later PR.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,89 @@
...
+[![Sponsor with Open Collective](https://opencollective.com/hypha/tiers/backer.svg?avatarHeight=75&width=600)](https://opencollective.com/hypha)
+
+[![Find and sponsor issues](https://issuehunt.io/static/embed/issuehunt-button-v1.svg)](https://issuehunt.io/r/OpenTechFund/hypha)
 
-Submission management software for open calls
 
 This is the repo for the [Hypha site](https://www.hypha.app/).
 
-[Hypha code](https://github.com/OpenTechFund/hypha)
+[Hypha code](https://github.com/HyphaApp/hypha)
```
Comment (frjo) [inline README.md]: This will be true some time in the future but for now the main repo is OTF:s.

File: content/_index.md [hunk 2]
```diff
@@ -1,86 +1,137 @@
...
+
+Hypha [source code is hosted on Github](https://github.com/HyphaApp/) and [development takes](https://github.com/HyphaApp/hypha/issues) place in the open.
+
+Hypha uses the [BSD 3-clause](./LICENSE) license.
+
+[![<OpenTechFund>](https://circleci.com/gh/OpenTechFund/hypha.svg?style=shield)](https://circleci.com/gh/OpenTechFund/hypha)
+
+## Deploying Hypha
+
+To support organisations having ownership over their data, Hypha can be deployed [on Heroku](https://github.com/OpenTechFund/hypha/wiki/Deployment:-Heroku), [using Docker](https://github.com/OpenTechFund/hypha/wiki/Deployment:-Docker), or [ with your own server](https://github.com/OpenTechFund/hypha/wiki/Deployment:-Standalone-App).
```
Comment (frjo) [inline content/_index.md]: The extra space before "with" in `[ with your own` should be removed.

**Review by frjo — CHANGES_REQUESTED** [review]
Looks good with the very minor things I noted.

File: README.md (hunk 1 above)
Comment (ei8fdb) [reply README.md]: Sure. I was wondering about that. I'll change it back now.

File: content/_index.md (hunk 2 above)
Comment (ei8fdb) [reply content/_index.md]: 👍

**Review by ei8fdb — COMMENTED** [review]
@frjo ready.

**Review by frjo — APPROVED** [review]

**Pull request merged**